web/Weblate-server.qmd

```diff
@@ -233,6 +275,8 @@ Weblate will provide a suggestion on how to fix, but in short:
     git rebase --continue
     ```
 
+    I usually do the `git` commands inside Docker (at `app/data/vcs/r-project/base-r-gui`), and the edits outside of Docker (at `/var/lib/docker/volumes/weblate-docker_weblate-data/_data/vcs/r-project/base-r-gui`) as it's more comfortable, but feel free to install anything that makes your life easier even within Docker (see e.g. `po3way` above).
```

Comment:
> install anything

IIUC that requires attaching with `-u root` is that right? Might be worth emphasizing if so.

Reply:
yeah, I thought that the `po3way` example reference might be enough .. but you are right, better to be explicit :+1: